src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Circle extends Figure implements AreaCalculator {
+
+    private int radius;
+
+    public Circle(String color, String name, int radius) {
+        super(color, name);
+        this.radius = radius;
+    }
+
+
+    @Override
+    public int calculateArea() {
+        return radius * radius * (int) Math.PI;
```

Comment:
The area calculation is incorrect. Casting `Math.PI` to an `int` results in a loss of precision, as it truncates the decimal part. Consider using `Math.PI` as a `double` and returning a `double` for more accurate area calculation.